src/tarantool-client/Converters/ReflectionConverter.cs

```diff
@@ -7,30 +7,37 @@
 // ReSharper disable once RedundantUsingDirective
 using System.Reflection;
 
-using TarantoolDnx.MsgPack;
+using MsgPack.Light;
 
 namespace tarantool_client.Converters
 {
     public class ReflectionConverter : IMsgPackConverter<object>
     {
-        public void Write(object value, IMsgPackWriter writer, MsgPackContext context)
+        private MsgPackContext _context;
```

Comment:
Actually, you should ask that question. That converter can be useful for example in non-typed Select method overload. But if that scenario will not be used, we can remove that converter, and some methods overloadings.